### DISCUSSION TITLE
Update or Fix needed

### DISCUSSION THREAD

**Discussion by zorglube** [CleverCloud/Community#49] [Add-ons]
Just after instanciation of a Matomo Add-On on Clever, at the first access of the Matomo IHM I see two error messages : 

- `WARNING: /plugins/LoginOIDC/Controller.php(388): Warning - Undefined variable $result - Matomo 5.1.0 - Please report this message in the Matomo forums: https://forum.matomo.org (please do a search first as it might have been reported already) (Module: LoginOIDC, Action: callback, In CLI mode: false)`
- `WARNING: Could not send Piwik\Plugins\CoreAdminHome\Emails\UserCreatedEmail email: /vendor/phpmailer/phpmailer/src/PHPMailer.php(1101): Adresse courriel non valide : (to): [Query: ?module=LoginOIDC&amp;action=callback&amp;provider=oidc&amp;cc_token=******&amp;state=2133f93648af1f5d48bbeec72a7b3d7d&amp;session_state=aaea8c3a-2a06-43dc-8ee5-2a80a52da12f&amp;code=a967 (Module: LoginOIDC, Action: callback, In CLI mode: false)`

![2024-12-19 13 17 56 qvfjshcweffsvuhxdjfd-matomo services clever-cloud com 32cfbd3499a8](https://github.com/user-attachments/assets/41e29878-2300-4485-b1ff-6a3370d12aee)

**Comment by davlgd** [comment]
Thanks for the feedback. We're working on a new release of Matomo based on `5.2` and upgraded deployment, I'll check this issue with team to fix it.
